### PR TITLE
Redirect links are not available on open software page #998

### PR DESCRIPTION
**Pull request by fnduister**
When looking at the open-source software or even any other open-source resources, we don't provide the link to redirect the user to the aforementioned resource.

### CODE REVIEW

File: _data/i18n/tables.yml [hunk 1]
```diff
@@ -87,3 +87,9 @@ lightbox:
   beingUsed:
     en: 'Is being used by:'
     fr: 'Est utilisé par :'
+  website:
+    en: 'Website:'
+    fr: 'Site internet:'
+  githubRepo:
```
Comment (ShadeWyrm) [inline _data/i18n/tables.yml]: Doesn't seem to be used anywhere? Also Github may not be the Repo used by any individual solution.

Comment (ShadeWyrm) [reply _data/i18n/tables.yml]: @fnduister

Comment (fnduister) [reply _data/i18n/tables.yml]: try I actually removed it for that reason, forgot to remove it from the table.

Comment (fnduister) [reply _data/i18n/tables.yml]: i could use repo only but the information doesn't seem to be available for every software

Comment (ShadeWyrm) [reply _data/i18n/tables.yml]: We don't ask for Repo at all anymore, so I suspect that's old data that never got scrubbed. I'd just remove it.

Comment (fnduister) [reply _data/i18n/tables.yml]: it's removed

**Pull request merged**